src/common/preferences/preferences.html

```diff
@@ -35,53 +35,51 @@
 				<div class="pane-image"><img src="../images/prefs-general.png" alt=""/></div>
 				<div class="pane-label">General</div>
 			</div>
+			<div id="pane-proxies" class="pane" style="display: none">
+				<div class="pane-image"><img src="../images/prefs-proxies.png" alt=""/></div>
+				<div class="pane-label">Proxies</div>
+			</div>
 			<div id="pane-advanced" class="pane">
 				<div class="pane-image"><img src="../images/prefs-advanced.png" alt=""/></div>
 				<div class="pane-label">Advanced</div>
 			</div>
 		</div>
 		<div id="content-general" class="content">
 			<div class="group">
-				<div class="group-title">Zotero Standalone</div>
+				<div class="group-title">Zotero Status</div>
 				<div class="group-content">
-					<p>Zotero Standalone is currently <span id="general-span-standalone-status"></span>.</p>
+					<p>Zotero is currently <span id="general-span-standalone-status"></span>.</p>
 					<p><input type="button" value="Update Status" id="general-button-update-standalone-status"/></p>
 				</div>
 			</div>
 			<div class="group">
-				<div class="group-title">Save via Server</div>
+				<div class="group-title">Save to Zotero.org</div>
```

Comment:
> given that people don't use or understand domain names anymore

They don't? What do people use then?

Judging from the few people that wander into forums every now and then attempting to use Zotero Connectors without realizing there is a client and checking their library on the website, "Save to zotero.org" sounds like the most user friendly name.

For those syncing, who may need to use the connector without the client for whatever reason, "Save to Web Library" may be a little clearer, but "Save to zotero.org" still works. "Save to Server" is confusing. The immediate questions are "What server? Where is said server? Is there a user-friendly way to look at what is saved on this server?"